st2actions/st2actions/container/base.py

```diff
@@ -93,10 +95,25 @@ def _do_run(self, runner, runnertype_db, action_db, liveaction_db):
             runner_params, action_params = param_utils.get_finalized_params(
                 runnertype_db.runner_parameters, action_db.parameters, liveaction_db.parameters)
             runner.runner_parameters = runner_params
-            LOG.debug('Performing pre-run for runner: %s', runner)
+
+            LOG.debug('Performing pre-run for runner: %s', runner.runner_id)
             runner.pre_run()
 
-            LOG.debug('Performing run for runner: %s', runner)
+            # Mask secret parameters in the log context
+            runner_parameters_specs = runnertype_db.runner_parameters
+            action_parameters_sepcs = action_db.parameters
+
+            secret_runner_parameters = get_secret_parameters(parameters=runner_parameters_specs)
```

Comment:
Just FYI - I've refactored this and made the code cleaner - cb7c3154cd4fda10cb7272b2771b7646321c9ff8.

Now all the masking code is encapsulated inside the `mask_secrets` method and it uses existing masking abstraction / framework.